### PR TITLE
Add unit tests for periphrastic passive voice helper files

### PR DESCRIPTION
**Pull request by FAMarfuaty**
## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Creates a new passiveVoice/periphrastic folder in yoastseo/spec/languageProcessing/helpers and adds the spec files for periphrastic helpers there. Also improves code styles of the periphrastic helper files and Dutch `getParticiple` helper file.

## Relevant technical choices:

* The unit tests for `checkException` file are not added since we stil plan to refactor this file further. This function is an integral part of a class and doesn't make much sense on its own.

## Test instructions

This PR can be tested by following these steps:

*

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes
* [ ] This PR changes the UI in the plugin. I have added the 'UI change' label to this PR.

## Quality assurance

* [ ] I have tested this code to the best of my abilities
* [ ] I have added unittests to verify the code works as intended

Fixes #https://yoast.atlassian.net/browse/LINGO-564

### CODE REVIEW

File: packages/yoastseo/spec/languageProcessing/helpers/passiveVoice/periphrastic/getSentencePartsSpec.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+};
+
+describe( "splits sentences into parts", function() {
+	it( "returns the sentence parts that are passive", function() {
+		const sentence =  "He is banned from the cafe.";
+		expect( getSentenceParts( sentence, options1 )[ 0 ].getSentencePartText() ).toBe( "is banned from the cafe." );
+		expect( getSentenceParts( sentence, options1 ).length ).toBe( 1 );
+	} );
+	it( "filters out sentence parts without auxiliary", function() {
+		const sentence = "The English are always throwing parties.";
```
Comment (manuelaugustin) [inline packages/yoastseo/spec/languageProcessing/helpers/passiveVoice/periphrastic/getSentencePartsSpec.js:48]: The description says that sentence parts without auxiliary are filtered out. However, specs only test whether there is a sentence part. It doesn't say anything about whether a sentence part is filtered when it's not passive. For that, you'd have to check something like:
`expect( getSentenceParts( sentence, options1 )[ 0 ].isPassive() ).toBe( false );`

File: packages/yoastseo/spec/languageProcessing/helpers/passiveVoice/periphrastic/getSentencePartsSpec.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+		const sentence = "\"This is it\", he said.";
+		expect( getSentenceParts( sentence, options1 )[ 0 ].getSentencePartText() ).toBe( "is it\"" );
+		expect( getSentenceParts( sentence, options1 ).length ).toBe( 1 );
+	} );
+	it( "doesn't return sentence parts when an auxiliary is followed by a word from the followingAuxiliaryExceptionWords list", function() {
+		// Exception word after auxiliary: el.
+		const sentence = "Es el capítulo preferido de varios miembros del equipo de producción.";
+		expect( getSentenceParts( sentence, options1 ).length ).toBe( 0 );
+	} );
+	it( "returns the sentence parts that are passive", function() {
```
Comment (manuelaugustin) [inline packages/yoastseo/spec/languageProcessing/helpers/passiveVoice/periphrastic/getSentencePartsSpec.js:95]: Also here it's good to check that the sentence part returned is actually passive.

**Review by manuelaugustin — CHANGES_REQUESTED** [review]
CR 🚧

Looks great overall! There just were a few small points, see comments.

**Comment by FAMarfuaty** [comment]
@manuelaugustin I've adjusted the specs based on your suggestion.

**Comment by manuelaugustin** [comment]
CR2 👍

**Pull request merged**